requirements.txt

```diff
@@ -12,5 +12,5 @@
 #   See the License for the specific language governing permissions and
 #   limitations under the License.
 
-mypy-protobuf==3.2
-grpcio-tools==1.50.0
\ No newline at end of file
+mypy-protobuf==3.4
+grpcio-tools==1.56.0
```

Comment:
Does it match grpc-common dependency? If not we will have problems when installing both

Reply:
Yes, https://github.com/th2-net/th2-grpc-common/pull/56/files#diff-60f61ab7a8d1910d86d9fda2261620314edcae5894d5aaa236b821c7256badd7R124